### PR TITLE
Add DoppelGANger to bibliography

### PR DESCRIPTION
**Pull request by fjxmlzn**

### CODE REVIEW

File: bibliography.bib [hunk 1]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
 
 ################ 2020
 
+@INPROCEEDINGS{clusterdata:Lin2020,
+   title = {Using GANs for Sharing Networked Time Series Data: Challenges, Initial Promise, and Open Questions},
```
Comment (johnwilkes) [inline bibliography.bib:550]: s/GANs/{GANs}/ to keep the capitalization correct.

File: bibliography.bib [hunk 2]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
...
+   author = {Lin, Zinan and Jain, Alankar and Wang, Chen and Fanti, Giulia and Sekar, Vyas},
+   year = {2020},
+   isbn = {9781450381383},
+   publisher = {Association for Computing Machinery},
+   address = {New York, NY, USA},
```
Comment (johnwilkes) [inline bibliography.bib:555]: Please use this for the address of the conference, not ACM.  Given that it was virtual, I'd just omit this. Thx.

File: bibliography.bib [hunk 3]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
...
...tasets with minimal expert knowledge. As a specific target, our focus in this paper is on time series datasets with metadata (e.g., packet loss rate measurements with corresponding ISPs). We identify key challenges of existing GAN approaches for such workloads with respect to fidelity (e.g., long-term dependencies, complex multidimensional relationships, mode collapse) and privacy (i.e., existing guarantees are poorly understood and can sacrifice fidelity). To improve fidelity, we design a custom workflow called DoppelGANger (DG) and demonstrate that across diverse real-world datasets (e.g., bandwidth measurements, cluster requests, web sessions) and use cases (e.g., structural characterization, predictive modeling, algorithm comparison), DG achieves up to 43% better fidelity than baseline models. Although we do not resolve the privacy problem in this work, we identify fundamental challenges with both classical notions of privacy and recent advances to improve the privacy properties of GANs, and suggest a potential roadmap for addressing these challenges. By shedding light on the promise and challenges, we hope our work can rekindle the conversation on workflows for data sharing.},
```
Comment (johnwilkes) [inline bibliography.bib:558]: Please break the lines, as is done in the other entries.  thx.

File: bibliography.bib [hunk 4]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
...
+   booktitle = {Proceedings of the ACM Internet Measurement Conference},
+   pages = {464–483},
```
Comment (johnwilkes) [inline bibliography.bib:560]: s/-/--/ to get an en-dash

File: bibliography.bib [hunk 5]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
...
+   numpages = {20},
+   keywords = {privacy, synthetic data generation, time series, generative adversarial networks},
+   location = {Virtual Event, USA},
+   series = {IMC '20}
```
Comment (johnwilkes) [inline bibliography.bib:564]: I usually prefer to put this into the booktitle like this:

booktitle = {Proceedings of the ACM Internet Measurement Conference (IMC 2020)},

File: bibliography.bib [hunk 6]
```diff
@@ -546,6 +546,24 @@ @Article{clusterdata:Mishra2010
...
+   booktitle = {Proceedings of the ACM Internet Measurement Conference},
+   pages = {464–483},
+   numpages = {20},
+   keywords = {privacy, synthetic data generation, time series, generative adversarial networks},
+   location = {Virtual Event, USA},
```
Comment (johnwilkes) [inline bibliography.bib:563]: Let's just skip this. Thx.

**Review by johnwilkes — CHANGES_REQUESTED** [review]
Thank you so much for submitting this as a pull request!

I've asked for a few changes to make this match the style of the other entries.  Hope that's OK.
  john

File: bibliography.bib (hunk 1 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

File: bibliography.bib (hunk 2 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

File: bibliography.bib (hunk 3 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

File: bibliography.bib (hunk 4 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

File: bibliography.bib (hunk 5 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

File: bibliography.bib (hunk 6 above)
Comment (fjxmlzn) [reply bibliography.bib]: Done.

**Comment by fjxmlzn** [comment]
Hi John,

Thanks for the suggestions! I have fixed all of the issues. Let me know if there is anything else I should fix.

Zinan

**Review by johnwilkes — APPROVED** [review]
thanks!

**Comment by johnwilkes** [comment]
Thanks.  It's not quite right (see the different indentations), but I
suppose that's OK.

**Pull request merged**